src/main/java/hwjdi/beans/MetalColorEntity.java

```diff
@@ -0,0 +1,166 @@
+package hwjdi.beans;
+
+import com.epam.jdi.tools.DataClass;
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "summary",
+        "elements",
+        "color",
+        "metals",
+        "vegetables"
+})
+public class MetalColorEntity extends DataClass<MetalColorEntity> {
```

Comment:
Is this file auto generated ?
If yes, it should not be in git at all. If no, it can be simplified, you should not add unnecessary annotations here.

Reply:
Yes, it was. Got this. Removed autogeneration, simplified class.